app/controllers/games_controller.rb

```diff
@@ -31,16 +33,16 @@ def player_passcode
         )
 
         while game.attempts.count < ChancesAndGuesses::CHANCES
-          guess = ValidColor.passcode
+          @guess = MiniMax.new(passcode: passcode_colors).play
 
           Attempt.create!(
             game: game,
             player: computer_player,
-            values: guess
+            values: @guess
           )
 
-          if guess == passcode
-            break
+          if @guess == passcode
+            "break"
```

Comment:
This is a strange change. Instead of getting out of a loop, it returns a string. Check if that's intended, otherwise maybe explain why is this?

Reply:
This is not intentional. I mistakenly changed it when I was trying to implement the minimax algorithm. I will revert it.